tests/integration/test_gitlab.py

```diff
@@ -0,0 +1,162 @@
+import os
+import unittest
+
+from ogr.services.gitlab import GitlabService
+from ogr.persistent_storage import PersistentObjectStorage
+
+DATA_DIR = "test_data"
+PERSISTENT_DATA_PREFIX = os.path.join(
+    os.path.dirname(os.path.realpath(__file__)), DATA_DIR
+)
+
+
+class GitlabTests(unittest.TestCase):
+    def setUp(self):
+        self.token = os.environ.get("GITLAB_TOKEN")
+        self.user = os.environ.get("GITLAB_USER")
+        test_name = self.id() or "all"
+
+        persistent_data_file = os.path.join(
+            PERSISTENT_DATA_PREFIX, f"test_gitlab_data_{test_name}.yaml"
+        )
+        PersistentObjectStorage().storage_file = persistent_data_file
+
+        if PersistentObjectStorage().is_write_mode and (
+            not self.user or not self.token
+        ):
+            raise EnvironmentError("please set GITLAB_TOKEN GITLAB_USER env variables")
+
+        self.service = GitlabService(
+            token=self.token, url="https://gitlab.cee.redhat.com"
```

Comment:
This is our internal instance. (It's not accessible from outside and we should not use internal links outside of our network...)

Please, use some easily-accessible instance (e.g. gitlab.com or gitlab.gnome.org).